net/sftp-server/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sftp-server
+PKG_VERSION:=3
```

Comment:
btw this look weird. Is it taken from upstream?

Reply:
I took it from here: https://github.com/ewxrjk/sftpserver/blob/master/CHANGES.md
That's why it is supposed to be version 3. Let me know which one is better. I've now set it to version 2.